### PR TITLE
xfail 2 tests using scikit-learn for DBSCAN

### PR DESCRIPTION
**Pull request by jGaboardi**
This PR provides a temporary solution to #219 until we figure out what's going on with [`scikit-learn==1.1.3`](https://github.com/pysal/esda/actions/runs/3457868371/jobs/5771724315#step:5:215).

### CODE REVIEW

**Comment by martinfleis** [comment]
Strictly speaking, we do not require `==1.0.2` but `<1.1.3` or `<1.1` don't we?

**Comment by jGaboardi** [comment]
> Strictly speaking, we do not require `==1.0.2` but `<1.1.3` or `<1.1` don't we?

Good point. Will update.

File: environment.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ channels:
...
+  - geopandas
   - libpysal
-  - scikit-learn
   - mapclassify
-  - geopandas
-  - numba
   - matplotlib
+  - numba
+  - pandas
+  - scikit-learns<1.1
```
Comment (martinfleis) [inline environment.yml:12]:
```suggestion
  - scikit-learn<1.1
```

typo

Comment (jGaboardi) [reply environment.yml:12]: Yikes. Good catch.

**Review by martinfleis — APPROVED** [review]
We should also look into the cause of the failure in 1.1, this is not a sustainable solution.

**Comment by jGaboardi** [comment]
> We should also look into the cause of the failure in 1.1, this is not a sustainable solution.

Much agreed. This is merely to get CI green again for modernization efforts so two failing tests don't completely stop development.

**Comment by jGaboardi** [comment]
> We should also look into the cause of the failure in 1.1, this is not a sustainable solution.

Much agreed. This is merely to get CI green again for modernization efforts so two failing tests don't completely stop development.

**Comment by martinfleis** [comment]
> This is merely to get CI green again for modernization efforts so two failing tests don't completely stop development.

Wouldn't a better solution be an `xfail` mark then?

**Comment by jGaboardi** [comment]
> Wouldn't a better solution be an `xfail` mark then?

Probably, yes. Will update accordingly.

File: environment.yml [hunk 2]
```diff
@@ -3,11 +3,12 @@ channels:
...
   - libpysal
-  - scikit-learn
   - mapclassify
-  - geopandas
-  - numba
   - matplotlib
+  - numba
+  - pandas
+  - scikit-learn
+  - scikit-learn<1.1
```
Comment (martinfleis) [inline environment.yml:13]: Do we need this then?
```suggestion
```

Comment (jGaboardi) [reply environment.yml:13]: Nope, that snuck back in during pull/merge.

**Pull request merged**